code/Forms/GridFieldPrintButtonExtension.php

```diff
@@ -11,6 +11,6 @@ class GridFieldPrintButtonExtension extends Extension
     public function updatePrintData($data)
     {
         $module = ModuleLoader::getModule('silverstripe/admin');
-        Requirements::css($module->getResourcePath('client/dist/styles/GridField_print.css'));
+        Requirements::css($module->getRelativeResourcePath('client/dist/styles/GridField_print.css'));
```

Comment:
how come it needs to be a relative link?

Reply:
Because otherwise you get the full filesystem path.

```html
<style href="/Users/dmooyman/Sites/ss40test/admin/client/dist/styles/GridField_print.css" >
```